### DISCUSSION TITLE
HII EVERYONE !!

### DISCUSSION THREAD

**Discussion by aakash1233333** [wagtail/gsoc#15] [General]
HII AAKASH THIS SIDE .WAS GOING THROUGH THE GSOC'23 ORGANIZATION WEBSITE FOUND WAGTAIL.CAN ANYBODY PLEASE BRIEF ABOUT THIS ORGANIZATION & CAN TELL ME ABOUT THE MAIN PROJECT THAT IS UNDER GSOC '23

**Comment by thibaudcolas** [comment]
Hi @aakash1233333, please take a look at our [contributor guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md). You’ll have to read it for yourself, we can’t just hand everyone over information separately.
